[2024-03-05T20:54:11.183+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: citibikedataset.download_dataset scheduled__2023-04-01T00:00:00+00:00 [queued]>
[2024-03-05T20:54:11.188+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: citibikedataset.download_dataset scheduled__2023-04-01T00:00:00+00:00 [queued]>
[2024-03-05T20:54:11.188+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2024-03-05T20:54:11.195+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): download_dataset> on 2023-04-01 00:00:00+00:00
[2024-03-05T20:54:11.198+0000] {standard_task_runner.py:60} INFO - Started process 240 to run task
[2024-03-05T20:54:11.201+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'citibikedataset', 'download_dataset', 'scheduled__2023-04-01T00:00:00+00:00', '--job-id', '13', '--raw', '--subdir', 'DAGS_FOLDER/dag_citibikenyc.py', '--cfg-path', '/tmp/tmpbdo_gcma']
[2024-03-05T20:54:11.202+0000] {standard_task_runner.py:88} INFO - Job 13: Subtask download_dataset
[2024-03-05T20:54:11.228+0000] {task_command.py:423} INFO - Running <TaskInstance: citibikedataset.download_dataset scheduled__2023-04-01T00:00:00+00:00 [running]> on host 8277132cfd53
[2024-03-05T20:54:11.257+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/context.py:207 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2024-03-05T20:54:11.274+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='felipeparfitt' AIRFLOW_CTX_DAG_ID='citibikedataset' AIRFLOW_CTX_TASK_ID='download_dataset' AIRFLOW_CTX_EXECUTION_DATE='2023-04-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-04-01T00:00:00+00:00'
[2024-03-05T20:54:11.275+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-03-05T20:54:11.276+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'sh ${AIRFLOW_HOME}/include/download_bike_dataset.sh 2023 04']
[2024-03-05T20:54:11.281+0000] {subprocess.py:86} INFO - Output:
[2024-03-05T20:54:11.282+0000] {subprocess.py:93} INFO - 
[2024-03-05T20:54:11.282+0000] {subprocess.py:93} INFO - Downloading citibike dataset from 2023/04
[2024-03-05T20:54:11.283+0000] {subprocess.py:93} INFO - /opt/***/include/dataset/raw/2023/04
[2024-03-05T20:54:11.284+0000] {subprocess.py:93} INFO - /opt/***/include/download_bike_dataset.sh: 14: wget: Permission denied
[2024-03-05T20:54:11.285+0000] {subprocess.py:97} INFO - Command exited with return code 1
[2024-03-05T20:54:11.291+0000] {taskinstance.py:2698} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 433, in _execute_task
    result = execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/bash.py", line 212, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2024-03-05T20:54:11.293+0000] {taskinstance.py:1138} INFO - Marking task as UP_FOR_RETRY. dag_id=citibikedataset, task_id=download_dataset, execution_date=20230401T000000, start_date=20240305T205411, end_date=20240305T205411
[2024-03-05T20:54:11.299+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 13 for task download_dataset (Bash command failed. The command returned a non-zero exit code 1.; 240)
[2024-03-05T20:54:11.333+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2024-03-05T20:54:11.345+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-03-05T21:12:40.638+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: citibikedataset.download_dataset scheduled__2023-04-01T00:00:00+00:00 [queued]>
[2024-03-05T21:12:40.643+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: citibikedataset.download_dataset scheduled__2023-04-01T00:00:00+00:00 [queued]>
[2024-03-05T21:12:40.643+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2024-03-05T21:12:40.650+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): download_dataset> on 2023-04-01 00:00:00+00:00
[2024-03-05T21:12:40.653+0000] {standard_task_runner.py:60} INFO - Started process 778 to run task
[2024-03-05T21:12:40.655+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'citibikedataset', 'download_dataset', 'scheduled__2023-04-01T00:00:00+00:00', '--job-id', '43', '--raw', '--subdir', 'DAGS_FOLDER/dag_citibikenyc.py', '--cfg-path', '/tmp/tmp_wyy02v7']
[2024-03-05T21:12:40.656+0000] {standard_task_runner.py:88} INFO - Job 43: Subtask download_dataset
[2024-03-05T21:12:40.680+0000] {task_command.py:423} INFO - Running <TaskInstance: citibikedataset.download_dataset scheduled__2023-04-01T00:00:00+00:00 [running]> on host 8277132cfd53
[2024-03-05T21:12:40.703+0000] {abstractoperator.py:707} ERROR - Exception rendering Jinja template for task 'download_dataset', field 'bash_command'. Template: "sh ${AIRFLOW_HOME}/include/download_bike_dataset.sh {{ ds.strftime('%Y') }} {{ ds.strftime('%m') }}"
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/abstractoperator.py", line 699, in _do_render_template_fields
    rendered_content = self.render_template(
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 174, in render_template
    return self._render(template, context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/abstractoperator.py", line 654, in _render
    return super()._render(template, context, dag=dag)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 131, in _render
    return render_template_to_string(template, context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/utils/helpers.py", line 289, in render_template_to_string
    return render_template(template, cast(MutableMapping[str, Any], context), native=False)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/utils/helpers.py", line 284, in render_template
    return "".join(nodes)
  File "<template>", line 13, in root
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/sandbox.py", line 391, in call
    if not __self.is_safe_callable(__obj):
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/sandbox.py", line 275, in is_safe_callable
    getattr(obj, "unsafe_callable", False) or getattr(obj, "alters_data", False)
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/runtime.py", line 859, in __getattr__
    return self._fail_with_undefined_error()
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/runtime.py", line 852, in _fail_with_undefined_error
    raise self._undefined_exception(self._undefined_message)
jinja2.exceptions.UndefinedError: 'str object' has no attribute 'strftime'
[2024-03-05T21:12:40.704+0000] {taskinstance.py:2698} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 2334, in _run_raw_task
    self._execute_task_with_callbacks(context, test_mode, session=session)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 2465, in _execute_task_with_callbacks
    task_orig = self.render_templates(context=context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 2877, in render_templates
    original_task.render_template_fields(context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/baseoperator.py", line 1241, in render_template_fields
    self._do_render_template_fields(self, self.template_fields, context, jinja_env, set())
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/utils/session.py", line 79, in wrapper
    return func(*args, session=session, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/abstractoperator.py", line 699, in _do_render_template_fields
    rendered_content = self.render_template(
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 174, in render_template
    return self._render(template, context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/abstractoperator.py", line 654, in _render
    return super()._render(template, context, dag=dag)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 131, in _render
    return render_template_to_string(template, context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/utils/helpers.py", line 289, in render_template_to_string
    return render_template(template, cast(MutableMapping[str, Any], context), native=False)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/utils/helpers.py", line 284, in render_template
    return "".join(nodes)
  File "<template>", line 13, in root
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/sandbox.py", line 391, in call
    if not __self.is_safe_callable(__obj):
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/sandbox.py", line 275, in is_safe_callable
    getattr(obj, "unsafe_callable", False) or getattr(obj, "alters_data", False)
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/runtime.py", line 859, in __getattr__
    return self._fail_with_undefined_error()
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/runtime.py", line 852, in _fail_with_undefined_error
    raise self._undefined_exception(self._undefined_message)
jinja2.exceptions.UndefinedError: 'str object' has no attribute 'strftime'
[2024-03-05T21:12:40.708+0000] {taskinstance.py:1138} INFO - Marking task as UP_FOR_RETRY. dag_id=citibikedataset, task_id=download_dataset, execution_date=20230401T000000, start_date=20240305T211240, end_date=20240305T211240
[2024-03-05T21:12:40.714+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 43 for task download_dataset ('str object' has no attribute 'strftime'; 778)
[2024-03-05T21:12:40.747+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2024-03-05T21:12:40.760+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-03-06T13:27:01.900+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: citibikedataset.download_dataset scheduled__2023-04-01T00:00:00+00:00 [queued]>
[2024-03-06T13:27:01.904+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: citibikedataset.download_dataset scheduled__2023-04-01T00:00:00+00:00 [queued]>
[2024-03-06T13:27:01.905+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2024-03-06T13:27:01.911+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): download_dataset> on 2023-04-01 00:00:00+00:00
[2024-03-06T13:27:01.914+0000] {standard_task_runner.py:60} INFO - Started process 497 to run task
[2024-03-06T13:27:01.916+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'citibikedataset', 'download_dataset', 'scheduled__2023-04-01T00:00:00+00:00', '--job-id', '78', '--raw', '--subdir', 'DAGS_FOLDER/dag_citibikenyc.py', '--cfg-path', '/tmp/tmpaljiar5a']
[2024-03-06T13:27:01.917+0000] {standard_task_runner.py:88} INFO - Job 78: Subtask download_dataset
[2024-03-06T13:27:01.940+0000] {task_command.py:423} INFO - Running <TaskInstance: citibikedataset.download_dataset scheduled__2023-04-01T00:00:00+00:00 [running]> on host 45431f0a53e1
[2024-03-06T13:27:01.968+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/context.py:207 AirflowContextDeprecationWarning: Accessing 'next_execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_end' instead.
[2024-03-06T13:27:01.985+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='felipeparfitt' AIRFLOW_CTX_DAG_ID='citibikedataset' AIRFLOW_CTX_TASK_ID='download_dataset' AIRFLOW_CTX_EXECUTION_DATE='2023-04-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-04-01T00:00:00+00:00'
[2024-03-06T13:27:01.986+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-03-06T13:27:01.986+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'sh ${AIRFLOW_HOME}/include/download_bike_dataset.sh         2023 05']
[2024-03-06T13:27:01.991+0000] {subprocess.py:86} INFO - Output:
[2024-03-06T13:27:01.992+0000] {subprocess.py:93} INFO - 
[2024-03-06T13:27:01.992+0000] {subprocess.py:93} INFO - Downloading citibike dataset from 2023/05
[2024-03-06T13:27:01.997+0000] {subprocess.py:93} INFO -   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[2024-03-06T13:27:01.997+0000] {subprocess.py:93} INFO -                                  Dload  Upload   Total   Spent    Left  Speed
[2024-03-06T13:27:03.770+0000] {subprocess.py:93} INFO -   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0 25 3144k   25  798k    0     0   576k      0  0:00:05  0:00:01  0:00:04  576k100 3144k  100 3144k    0     0  1773k      0  0:00:01  0:00:01 --:--:-- 1773k
[2024-03-06T13:27:03.771+0000] {subprocess.py:93} INFO - 
[2024-03-06T13:27:03.771+0000] {subprocess.py:93} INFO - Download completed successfully!
[2024-03-06T13:27:03.771+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-03-06T13:27:03.783+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=citibikedataset, task_id=download_dataset, execution_date=20230401T000000, start_date=20240306T132701, end_date=20240306T132703
[2024-03-06T13:27:03.814+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-06T13:27:03.823+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-03-06T17:35:40.059+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: citibikedataset.download_dataset scheduled__2023-04-01T00:00:00+00:00 [queued]>
[2024-03-06T17:35:40.064+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: citibikedataset.download_dataset scheduled__2023-04-01T00:00:00+00:00 [queued]>
[2024-03-06T17:35:40.064+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2024-03-06T17:35:40.074+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): download_dataset> on 2023-04-01 00:00:00+00:00
[2024-03-06T17:35:40.077+0000] {standard_task_runner.py:60} INFO - Started process 1213 to run task
[2024-03-06T17:35:40.080+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'citibikedataset', 'download_dataset', 'scheduled__2023-04-01T00:00:00+00:00', '--job-id', '103', '--raw', '--subdir', 'DAGS_FOLDER/dag_citibikenyc.py', '--cfg-path', '/tmp/tmpg7o1lqod']
[2024-03-06T17:35:40.081+0000] {standard_task_runner.py:88} INFO - Job 103: Subtask download_dataset
[2024-03-06T17:35:40.107+0000] {task_command.py:423} INFO - Running <TaskInstance: citibikedataset.download_dataset scheduled__2023-04-01T00:00:00+00:00 [running]> on host e43ee8560afa
[2024-03-06T17:35:40.142+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/context.py:207 AirflowContextDeprecationWarning: Accessing 'next_execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_end' instead.
[2024-03-06T17:35:40.158+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='felipeparfitt' AIRFLOW_CTX_DAG_ID='citibikedataset' AIRFLOW_CTX_TASK_ID='download_dataset' AIRFLOW_CTX_EXECUTION_DATE='2023-04-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-04-01T00:00:00+00:00'
[2024-03-06T17:35:40.159+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-03-06T17:35:40.160+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'sh ${AIRFLOW_HOME}/include/download_bike_dataset.sh         2023 05']
[2024-03-06T17:35:40.165+0000] {subprocess.py:86} INFO - Output:
[2024-03-06T17:35:40.166+0000] {subprocess.py:93} INFO - 
[2024-03-06T17:35:40.166+0000] {subprocess.py:93} INFO - Downloading citibike dataset from 2023/05
[2024-03-06T17:35:40.171+0000] {subprocess.py:93} INFO -   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[2024-03-06T17:35:40.171+0000] {subprocess.py:93} INFO -                                  Dload  Upload   Total   Spent    Left  Speed
[2024-03-06T17:35:41.755+0000] {subprocess.py:93} INFO -   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0  0 3144k    0 25087    0     0  33331      0  0:01:36 --:--:--  0:01:36 33316100 3144k  100 3144k    0     0  1984k      0  0:00:01  0:00:01 --:--:-- 1985k
[2024-03-06T17:35:41.757+0000] {subprocess.py:93} INFO - Archive:  /opt/***/include/dataset/raw/2023/05/JC-202305-citibike-tripdata.csv.zip
[2024-03-06T17:35:41.831+0000] {subprocess.py:93} INFO -   inflating: /opt/***/include/dataset/raw/2023/05/JC-202305-citibike-tripdata.csv
[2024-03-06T17:35:41.831+0000] {subprocess.py:93} INFO -   inflating: /opt/***/include/dataset/raw/2023/05/__MACOSX/._JC-202305-citibike-tripdata.csv
[2024-03-06T17:35:41.833+0000] {subprocess.py:93} INFO - 	zip warning: name not matched: JC-202305-citibike-tripdata.csv
[2024-03-06T17:35:41.833+0000] {subprocess.py:93} INFO - 
[2024-03-06T17:35:41.833+0000] {subprocess.py:93} INFO - zip error: Nothing to do! (/opt/***/include/dataset/raw/2023/05/JC-202305-citibike-tripdata.csv.zip)
[2024-03-06T17:35:41.833+0000] {subprocess.py:93} INFO - 
[2024-03-06T17:35:41.833+0000] {subprocess.py:93} INFO - Download completed successfully!
[2024-03-06T17:35:41.833+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-03-06T17:35:41.845+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=citibikedataset, task_id=download_dataset, execution_date=20230401T000000, start_date=20240306T173540, end_date=20240306T173541
[2024-03-06T17:35:41.856+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-06T17:35:41.865+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-03-06T17:39:43.055+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: citibikedataset.download_dataset scheduled__2023-04-01T00:00:00+00:00 [queued]>
[2024-03-06T17:39:43.060+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: citibikedataset.download_dataset scheduled__2023-04-01T00:00:00+00:00 [queued]>
[2024-03-06T17:39:43.060+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2024-03-06T17:39:43.067+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): download_dataset> on 2023-04-01 00:00:00+00:00
[2024-03-06T17:39:43.070+0000] {standard_task_runner.py:60} INFO - Started process 1392 to run task
[2024-03-06T17:39:43.072+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'citibikedataset', 'download_dataset', 'scheduled__2023-04-01T00:00:00+00:00', '--job-id', '113', '--raw', '--subdir', 'DAGS_FOLDER/dag_citibikenyc.py', '--cfg-path', '/tmp/tmph3v3sn55']
[2024-03-06T17:39:43.074+0000] {standard_task_runner.py:88} INFO - Job 113: Subtask download_dataset
[2024-03-06T17:39:43.099+0000] {task_command.py:423} INFO - Running <TaskInstance: citibikedataset.download_dataset scheduled__2023-04-01T00:00:00+00:00 [running]> on host e43ee8560afa
[2024-03-06T17:39:43.129+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/context.py:207 AirflowContextDeprecationWarning: Accessing 'next_execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_end' instead.
[2024-03-06T17:39:43.147+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='felipeparfitt' AIRFLOW_CTX_DAG_ID='citibikedataset' AIRFLOW_CTX_TASK_ID='download_dataset' AIRFLOW_CTX_EXECUTION_DATE='2023-04-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-04-01T00:00:00+00:00'
[2024-03-06T17:39:43.148+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-03-06T17:39:43.148+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'sh ${AIRFLOW_HOME}/include/download_bike_dataset.sh         2023 05']
[2024-03-06T17:39:43.153+0000] {subprocess.py:86} INFO - Output:
[2024-03-06T17:39:43.155+0000] {subprocess.py:93} INFO - 
[2024-03-06T17:39:43.155+0000] {subprocess.py:93} INFO - Downloading citibike dataset from 2023/05
[2024-03-06T17:39:43.160+0000] {subprocess.py:93} INFO -   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[2024-03-06T17:39:43.160+0000] {subprocess.py:93} INFO -                                  Dload  Upload   Total   Spent    Left  Speed
[2024-03-06T17:39:44.771+0000] {subprocess.py:93} INFO -   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0  0 3144k    0 17004    0     0  21702      0  0:02:28 --:--:--  0:02:28 21688100 3144k  100 3144k    0     0  1951k      0  0:00:01  0:00:01 --:--:-- 1952k
[2024-03-06T17:39:44.774+0000] {subprocess.py:93} INFO - Archive:  /opt/***/include/dataset/raw/2023/05/JC-202305-citibike-tripdata.csv.zip
[2024-03-06T17:39:44.845+0000] {subprocess.py:93} INFO -   inflating: /opt/***/include/dataset/raw/2023/05/JC-202305-citibike-tripdata.csv
[2024-03-06T17:39:44.845+0000] {subprocess.py:93} INFO -   inflating: /opt/***/include/dataset/raw/2023/05/__MACOSX/._JC-202305-citibike-tripdata.csv
[2024-03-06T17:39:45.144+0000] {subprocess.py:93} INFO -   adding: opt/***/include/dataset/raw/2023/05/JC-202305-citibike-tripdata.csv (deflated 84%)
[2024-03-06T17:39:45.146+0000] {subprocess.py:93} INFO - 
[2024-03-06T17:39:45.146+0000] {subprocess.py:93} INFO - Download completed successfully!
[2024-03-06T17:39:45.146+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-03-06T17:39:45.159+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=citibikedataset, task_id=download_dataset, execution_date=20230401T000000, start_date=20240306T173943, end_date=20240306T173945
[2024-03-06T17:39:45.171+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-06T17:39:45.181+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-03-07T14:08:24.967+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: citibikedataset.download_dataset scheduled__2023-04-01T00:00:00+00:00 [queued]>
[2024-03-07T14:08:24.973+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: citibikedataset.download_dataset scheduled__2023-04-01T00:00:00+00:00 [queued]>
[2024-03-07T14:08:24.973+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2024-03-07T14:08:24.980+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): download_dataset> on 2023-04-01 00:00:00+00:00
[2024-03-07T14:08:24.983+0000] {standard_task_runner.py:60} INFO - Started process 1340 to run task
[2024-03-07T14:08:24.985+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'citibikedataset', 'download_dataset', 'scheduled__2023-04-01T00:00:00+00:00', '--job-id', '156', '--raw', '--subdir', 'DAGS_FOLDER/dag_citibikenyc.py', '--cfg-path', '/tmp/tmpngr74u7v']
[2024-03-07T14:08:24.987+0000] {standard_task_runner.py:88} INFO - Job 156: Subtask download_dataset
[2024-03-07T14:08:25.013+0000] {task_command.py:423} INFO - Running <TaskInstance: citibikedataset.download_dataset scheduled__2023-04-01T00:00:00+00:00 [running]> on host 0430cbdaac75
[2024-03-07T14:08:25.041+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/context.py:207 AirflowContextDeprecationWarning: Accessing 'next_execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_end' instead.
[2024-03-07T14:08:25.160+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='felipeparfitt' AIRFLOW_CTX_DAG_ID='citibikedataset' AIRFLOW_CTX_TASK_ID='download_dataset' AIRFLOW_CTX_EXECUTION_DATE='2023-04-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-04-01T00:00:00+00:00'
[2024-03-07T14:08:25.161+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-03-07T14:08:25.162+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'sh ${AIRFLOW_HOME}/include/download_bike_dataset.sh         2023 05']
[2024-03-07T14:08:25.168+0000] {subprocess.py:86} INFO - Output:
[2024-03-07T14:08:25.170+0000] {subprocess.py:93} INFO - 
[2024-03-07T14:08:25.170+0000] {subprocess.py:93} INFO - Downloading citibike dataset from 2023/05
[2024-03-07T14:08:25.175+0000] {subprocess.py:93} INFO -   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[2024-03-07T14:08:25.175+0000] {subprocess.py:93} INFO -                                  Dload  Upload   Total   Spent    Left  Speed
[2024-03-07T14:08:27.862+0000] {subprocess.py:93} INFO -   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0  0     0    0     0    0     0      0      0 --:--:--  0:00:01 --:--:--     0 12 3144k   12  377k    0     0   169k      0  0:00:18  0:00:02  0:00:16  169k100 3144k  100 3144k    0     0  1170k      0  0:00:02  0:00:02 --:--:-- 1169k
[2024-03-07T14:08:27.864+0000] {subprocess.py:93} INFO - Archive:  /opt/***/include/dataset/raw/2023/05/JC-202305-citibike-tripdata.csv.zip
[2024-03-07T14:08:27.937+0000] {subprocess.py:93} INFO -   inflating: /opt/***/include/dataset/raw/2023/05/JC-202305-citibike-tripdata.csv
[2024-03-07T14:08:27.937+0000] {subprocess.py:93} INFO -   inflating: /opt/***/include/dataset/raw/2023/05/__MACOSX/._JC-202305-citibike-tripdata.csv
[2024-03-07T14:08:28.234+0000] {subprocess.py:93} INFO -   adding: opt/***/include/dataset/raw/2023/05/JC-202305-citibike-tripdata.csv (deflated 84%)
[2024-03-07T14:08:28.236+0000] {subprocess.py:93} INFO - 
[2024-03-07T14:08:28.236+0000] {subprocess.py:93} INFO - Download completed successfully!
[2024-03-07T14:08:28.236+0000] {subprocess.py:93} INFO - /opt/***/include/dataset/raw/2023/05/JC-202305-citibike-tripdata.csv.zip
[2024-03-07T14:08:28.236+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-03-07T14:08:28.250+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=citibikedataset, task_id=download_dataset, execution_date=20230401T000000, start_date=20240307T140824, end_date=20240307T140828
[2024-03-07T14:08:28.289+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-07T14:08:28.307+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-03-07T17:55:51.235+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: citibikedataset.download_dataset scheduled__2023-04-01T00:00:00+00:00 [queued]>
[2024-03-07T17:55:51.240+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: citibikedataset.download_dataset scheduled__2023-04-01T00:00:00+00:00 [queued]>
[2024-03-07T17:55:51.240+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2024-03-07T17:55:51.248+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): download_dataset> on 2023-04-01 00:00:00+00:00
[2024-03-07T17:55:51.252+0000] {standard_task_runner.py:60} INFO - Started process 485 to run task
[2024-03-07T17:55:51.254+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'citibikedataset', 'download_dataset', 'scheduled__2023-04-01T00:00:00+00:00', '--job-id', '217', '--raw', '--subdir', 'DAGS_FOLDER/dag_citibikenyc.py', '--cfg-path', '/tmp/tmpqcw0q8eh']
[2024-03-07T17:55:51.255+0000] {standard_task_runner.py:88} INFO - Job 217: Subtask download_dataset
[2024-03-07T17:55:51.281+0000] {task_command.py:423} INFO - Running <TaskInstance: citibikedataset.download_dataset scheduled__2023-04-01T00:00:00+00:00 [running]> on host a0c898e5643f
[2024-03-07T17:55:51.310+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/context.py:207 AirflowContextDeprecationWarning: Accessing 'next_execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_end' instead.
[2024-03-07T17:55:51.329+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='felipeparfitt' AIRFLOW_CTX_DAG_ID='citibikedataset' AIRFLOW_CTX_TASK_ID='download_dataset' AIRFLOW_CTX_EXECUTION_DATE='2023-04-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-04-01T00:00:00+00:00'
[2024-03-07T17:55:51.330+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-03-07T17:55:51.331+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'sh ${AIRFLOW_HOME}/include/download_bike_dataset.sh         2023 05']
[2024-03-07T17:55:51.337+0000] {subprocess.py:86} INFO - Output:
[2024-03-07T17:55:51.339+0000] {subprocess.py:93} INFO - 
[2024-03-07T17:55:51.339+0000] {subprocess.py:93} INFO - Downloading citibike dataset from 2023/05
[2024-03-07T17:55:51.344+0000] {subprocess.py:93} INFO -   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[2024-03-07T17:55:51.344+0000] {subprocess.py:93} INFO -                                  Dload  Upload   Total   Spent    Left  Speed
[2024-03-07T17:55:52.952+0000] {subprocess.py:93} INFO -   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0  5 3144k    5  168k    0     0   170k      0  0:00:18 --:--:--  0:00:18  170k100 3144k  100 3144k    0     0  1954k      0  0:00:01  0:00:01 --:--:-- 1955k
[2024-03-07T17:55:52.955+0000] {subprocess.py:93} INFO - Archive:  /opt/***/include/dataset/raw/2023/05/JC-202305-citibike-tripdata.csv.zip
[2024-03-07T17:55:53.029+0000] {subprocess.py:93} INFO -   inflating: /opt/***/include/dataset/raw/2023/05/JC-202305-citibike-tripdata.csv
[2024-03-07T17:55:53.030+0000] {subprocess.py:93} INFO -   inflating: /opt/***/include/dataset/raw/2023/05/__MACOSX/._JC-202305-citibike-tripdata.csv
[2024-03-07T17:55:53.328+0000] {subprocess.py:93} INFO -   adding: opt/***/include/dataset/raw/2023/05/JC-202305-citibike-tripdata.csv (deflated 84%)
[2024-03-07T17:55:53.330+0000] {subprocess.py:93} INFO - 
[2024-03-07T17:55:53.330+0000] {subprocess.py:93} INFO - Download completed successfully!
[2024-03-07T17:55:53.330+0000] {subprocess.py:93} INFO - /opt/***/include/dataset/raw/2023/05/JC-202305-citibike-tripdata.csv.zip
[2024-03-07T17:55:53.330+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-03-07T17:55:53.345+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=citibikedataset, task_id=download_dataset, execution_date=20230401T000000, start_date=20240307T175551, end_date=20240307T175553
[2024-03-07T17:55:53.393+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-07T17:55:53.414+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
